[2023-01-16 16:51:14 +0000] [17700] [DEBUG] Current configuration:
  config: /home/debian/smeet/gunicorn_conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/smeet/gunicorn.sock']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fb16ff1c2f0>
  on_reload: <function OnReload.on_reload at 0x7fb16ff1c400>
  when_ready: <function WhenReady.when_ready at 0x7fb16ff1c510>
  pre_fork: <function Prefork.pre_fork at 0x7fb16ff1c620>
  post_fork: <function Postfork.post_fork at 0x7fb16ff1c730>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fb16ff1c840>
  worker_int: <function WorkerInt.worker_int at 0x7fb16ff1c950>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fb16ff1ca60>
  pre_exec: <function PreExec.pre_exec at 0x7fb16ff1cb70>
  pre_request: <function PreRequest.pre_request at 0x7fb16ff1cc80>
  post_request: <function PostRequest.post_request at 0x7fb16ff1cd08>
  child_exit: <function ChildExit.child_exit at 0x7fb16ff1ce18>
  worker_exit: <function WorkerExit.worker_exit at 0x7fb16ff1cf28>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fb16ff380d0>
  on_exit: <function OnExit.on_exit at 0x7fb16ff381e0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 16:51:14 +0000] [17700] [INFO] Starting gunicorn 20.1.0
[2023-01-16 16:51:14 +0000] [17700] [DEBUG] Arbiter booted
[2023-01-16 16:51:14 +0000] [17700] [INFO] Listening at: unix:/home/debian/smeet/gunicorn.sock (17700)
[2023-01-16 16:51:14 +0000] [17700] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2023-01-16 16:51:14 +0000] [17703] [INFO] Booting worker with pid: 17703
[2023-01-16 16:51:14 +0000] [17703] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:51:14 +0000] [17703] [INFO] Worker exiting (pid: 17703)
[2023-01-16 16:51:14 +0000] [17704] [INFO] Booting worker with pid: 17704
[2023-01-16 16:51:14 +0000] [17704] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:51:14 +0000] [17704] [INFO] Worker exiting (pid: 17704)
[2023-01-16 16:51:14 +0000] [17700] [DEBUG] 2 workers
[2023-01-16 16:51:14 +0000] [17700] [WARNING] Worker with pid 17704 was terminated due to signal 15
[2023-01-16 16:51:14 +0000] [17700] [INFO] Shutting down: Master
[2023-01-16 16:51:14 +0000] [17700] [INFO] Reason: Worker failed to boot.
[2023-01-16 16:52:22 +0000] [17742] [DEBUG] Current configuration:
  config: /home/debian/smeet/gunicorn_conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/smeet/gunicorn.sock']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fc0bb4b32f0>
  on_reload: <function OnReload.on_reload at 0x7fc0bb4b3400>
  when_ready: <function WhenReady.when_ready at 0x7fc0bb4b3510>
  pre_fork: <function Prefork.pre_fork at 0x7fc0bb4b3620>
  post_fork: <function Postfork.post_fork at 0x7fc0bb4b3730>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fc0bb4b3840>
  worker_int: <function WorkerInt.worker_int at 0x7fc0bb4b3950>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fc0bb4b3a60>
  pre_exec: <function PreExec.pre_exec at 0x7fc0bb4b3b70>
  pre_request: <function PreRequest.pre_request at 0x7fc0bb4b3c80>
  post_request: <function PostRequest.post_request at 0x7fc0bb4b3d08>
  child_exit: <function ChildExit.child_exit at 0x7fc0bb4b3e18>
  worker_exit: <function WorkerExit.worker_exit at 0x7fc0bb4b3f28>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fc0bb4cf0d0>
  on_exit: <function OnExit.on_exit at 0x7fc0bb4cf1e0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 16:52:22 +0000] [17742] [INFO] Starting gunicorn 20.1.0
[2023-01-16 16:52:22 +0000] [17742] [DEBUG] Arbiter booted
[2023-01-16 16:52:22 +0000] [17742] [INFO] Listening at: unix:/home/debian/smeet/gunicorn.sock (17742)
[2023-01-16 16:52:22 +0000] [17742] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2023-01-16 16:52:22 +0000] [17745] [INFO] Booting worker with pid: 17745
[2023-01-16 16:52:22 +0000] [17745] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:52:22 +0000] [17745] [INFO] Worker exiting (pid: 17745)
[2023-01-16 16:52:22 +0000] [17746] [INFO] Booting worker with pid: 17746
[2023-01-16 16:52:22 +0000] [17742] [DEBUG] 2 workers
[2023-01-16 16:52:22 +0000] [17746] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:52:22 +0000] [17746] [INFO] Worker exiting (pid: 17746)
[2023-01-16 16:52:22 +0000] [17742] [WARNING] Worker with pid 17746 was terminated due to signal 15
[2023-01-16 16:52:22 +0000] [17742] [INFO] Shutting down: Master
[2023-01-16 16:52:22 +0000] [17742] [INFO] Reason: Worker failed to boot.
[2023-01-16 16:56:28 +0000] [17788] [DEBUG] Current configuration:
  config: /home/debian/smeet/gunicorn_conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/smeet/gunicorn.sock']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fe95c6602f0>
  on_reload: <function OnReload.on_reload at 0x7fe95c660400>
  when_ready: <function WhenReady.when_ready at 0x7fe95c660510>
  pre_fork: <function Prefork.pre_fork at 0x7fe95c660620>
  post_fork: <function Postfork.post_fork at 0x7fe95c660730>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fe95c660840>
  worker_int: <function WorkerInt.worker_int at 0x7fe95c660950>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fe95c660a60>
  pre_exec: <function PreExec.pre_exec at 0x7fe95c660b70>
  pre_request: <function PreRequest.pre_request at 0x7fe95c660c80>
  post_request: <function PostRequest.post_request at 0x7fe95c660d08>
  child_exit: <function ChildExit.child_exit at 0x7fe95c660e18>
  worker_exit: <function WorkerExit.worker_exit at 0x7fe95c660f28>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fe95c67c0d0>
  on_exit: <function OnExit.on_exit at 0x7fe95c67c1e0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 16:56:28 +0000] [17788] [INFO] Starting gunicorn 20.1.0
[2023-01-16 16:56:28 +0000] [17788] [DEBUG] Arbiter booted
[2023-01-16 16:56:28 +0000] [17788] [INFO] Listening at: unix:/home/debian/smeet/gunicorn.sock (17788)
[2023-01-16 16:56:28 +0000] [17788] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2023-01-16 16:56:28 +0000] [17792] [INFO] Booting worker with pid: 17792
[2023-01-16 16:56:28 +0000] [17792] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:56:28 +0000] [17792] [INFO] Worker exiting (pid: 17792)
[2023-01-16 16:56:28 +0000] [17793] [INFO] Booting worker with pid: 17793
[2023-01-16 16:56:28 +0000] [17788] [DEBUG] 2 workers
[2023-01-16 16:56:28 +0000] [17793] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:56:28 +0000] [17793] [INFO] Worker exiting (pid: 17793)
[2023-01-16 16:56:28 +0000] [17788] [WARNING] Worker with pid 17793 was terminated due to signal 15
[2023-01-16 16:56:28 +0000] [17788] [INFO] Shutting down: Master
[2023-01-16 16:56:28 +0000] [17788] [INFO] Reason: Worker failed to boot.
[2023-01-16 16:56:51 +0000] [17827] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/smeet/gunicorn.sock']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f938bf652f0>
  on_reload: <function OnReload.on_reload at 0x7f938bf65400>
  when_ready: <function WhenReady.when_ready at 0x7f938bf65510>
  pre_fork: <function Prefork.pre_fork at 0x7f938bf65620>
  post_fork: <function Postfork.post_fork at 0x7f938bf65730>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f938bf65840>
  worker_int: <function WorkerInt.worker_int at 0x7f938bf65950>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f938bf65a60>
  pre_exec: <function PreExec.pre_exec at 0x7f938bf65b70>
  pre_request: <function PreRequest.pre_request at 0x7f938bf65c80>
  post_request: <function PostRequest.post_request at 0x7f938bf65d08>
  child_exit: <function ChildExit.child_exit at 0x7f938bf65e18>
  worker_exit: <function WorkerExit.worker_exit at 0x7f938bf65f28>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f938bf810d0>
  on_exit: <function OnExit.on_exit at 0x7f938bf811e0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 16:56:51 +0000] [17827] [INFO] Starting gunicorn 20.1.0
[2023-01-16 16:56:51 +0000] [17827] [DEBUG] Arbiter booted
[2023-01-16 16:56:51 +0000] [17827] [INFO] Listening at: unix:/home/debian/smeet/gunicorn.sock (17827)
[2023-01-16 16:56:51 +0000] [17827] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2023-01-16 16:56:51 +0000] [17830] [INFO] Booting worker with pid: 17830
[2023-01-16 16:56:52 +0000] [17831] [INFO] Booting worker with pid: 17831
[2023-01-16 16:56:52 +0000] [17830] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:56:52 +0000] [17830] [INFO] Worker exiting (pid: 17830)
[2023-01-16 16:56:52 +0000] [17831] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:56:52 +0000] [17831] [INFO] Worker exiting (pid: 17831)
[2023-01-16 16:56:52 +0000] [17827] [DEBUG] 2 workers
[2023-01-16 16:57:39 +0000] [17859] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/smeet/gunicorn.sock']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f8db6cf42f0>
  on_reload: <function OnReload.on_reload at 0x7f8db6cf4400>
  when_ready: <function WhenReady.when_ready at 0x7f8db6cf4510>
  pre_fork: <function Prefork.pre_fork at 0x7f8db6cf4620>
  post_fork: <function Postfork.post_fork at 0x7f8db6cf4730>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f8db6cf4840>
  worker_int: <function WorkerInt.worker_int at 0x7f8db6cf4950>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f8db6cf4a60>
  pre_exec: <function PreExec.pre_exec at 0x7f8db6cf4b70>
  pre_request: <function PreRequest.pre_request at 0x7f8db6cf4c80>
  post_request: <function PostRequest.post_request at 0x7f8db6cf4d08>
  child_exit: <function ChildExit.child_exit at 0x7f8db6cf4e18>
  worker_exit: <function WorkerExit.worker_exit at 0x7f8db6cf4f28>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f8db6d100d0>
  on_exit: <function OnExit.on_exit at 0x7f8db6d101e0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 16:57:39 +0000] [17859] [INFO] Starting gunicorn 20.1.0
[2023-01-16 16:57:39 +0000] [17859] [DEBUG] Arbiter booted
[2023-01-16 16:57:39 +0000] [17859] [INFO] Listening at: unix:/home/debian/smeet/gunicorn.sock (17859)
[2023-01-16 16:57:39 +0000] [17859] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2023-01-16 16:57:39 +0000] [17862] [INFO] Booting worker with pid: 17862
[2023-01-16 16:57:39 +0000] [17862] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:57:39 +0000] [17862] [INFO] Worker exiting (pid: 17862)
[2023-01-16 16:57:39 +0000] [17863] [INFO] Booting worker with pid: 17863
[2023-01-16 16:57:39 +0000] [17863] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:57:39 +0000] [17863] [INFO] Worker exiting (pid: 17863)
[2023-01-16 16:57:39 +0000] [17859] [WARNING] Worker with pid 17863 was terminated due to signal 15
[2023-01-16 16:57:39 +0000] [17859] [INFO] Shutting down: Master
[2023-01-16 16:57:39 +0000] [17859] [INFO] Reason: Worker failed to boot.
[2023-01-16 16:57:59 +0000] [17891] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/smeet/gunicorn.sock']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f2ce05ab2f0>
  on_reload: <function OnReload.on_reload at 0x7f2ce05ab400>
  when_ready: <function WhenReady.when_ready at 0x7f2ce05ab510>
  pre_fork: <function Prefork.pre_fork at 0x7f2ce05ab620>
  post_fork: <function Postfork.post_fork at 0x7f2ce05ab730>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f2ce05ab840>
  worker_int: <function WorkerInt.worker_int at 0x7f2ce05ab950>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f2ce05aba60>
  pre_exec: <function PreExec.pre_exec at 0x7f2ce05abb70>
  pre_request: <function PreRequest.pre_request at 0x7f2ce05abc80>
  post_request: <function PostRequest.post_request at 0x7f2ce05abd08>
  child_exit: <function ChildExit.child_exit at 0x7f2ce05abe18>
  worker_exit: <function WorkerExit.worker_exit at 0x7f2ce05abf28>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f2ce05c70d0>
  on_exit: <function OnExit.on_exit at 0x7f2ce05c71e0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 16:57:59 +0000] [17891] [INFO] Starting gunicorn 20.1.0
[2023-01-16 16:57:59 +0000] [17891] [DEBUG] Arbiter booted
[2023-01-16 16:57:59 +0000] [17891] [INFO] Listening at: unix:/home/debian/smeet/gunicorn.sock (17891)
[2023-01-16 16:57:59 +0000] [17891] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2023-01-16 16:57:59 +0000] [17894] [INFO] Booting worker with pid: 17894
[2023-01-16 16:57:59 +0000] [17894] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:57:59 +0000] [17894] [INFO] Worker exiting (pid: 17894)
[2023-01-16 16:57:59 +0000] [17891] [INFO] Shutting down: Master
[2023-01-16 16:57:59 +0000] [17891] [INFO] Reason: Worker failed to boot.
[2023-01-16 16:59:41 +0000] [17954] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/smeet/gunicorn.sock']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f14d8a8a2f0>
  on_reload: <function OnReload.on_reload at 0x7f14d8a8a400>
  when_ready: <function WhenReady.when_ready at 0x7f14d8a8a510>
  pre_fork: <function Prefork.pre_fork at 0x7f14d8a8a620>
  post_fork: <function Postfork.post_fork at 0x7f14d8a8a730>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f14d8a8a840>
  worker_int: <function WorkerInt.worker_int at 0x7f14d8a8a950>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f14d8a8aa60>
  pre_exec: <function PreExec.pre_exec at 0x7f14d8a8ab70>
  pre_request: <function PreRequest.pre_request at 0x7f14d8a8ac80>
  post_request: <function PostRequest.post_request at 0x7f14d8a8ad08>
  child_exit: <function ChildExit.child_exit at 0x7f14d8a8ae18>
  worker_exit: <function WorkerExit.worker_exit at 0x7f14d8a8af28>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f14d8aa60d0>
  on_exit: <function OnExit.on_exit at 0x7f14d8aa61e0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 16:59:41 +0000] [17954] [INFO] Starting gunicorn 20.1.0
[2023-01-16 16:59:41 +0000] [17954] [DEBUG] Arbiter booted
[2023-01-16 16:59:41 +0000] [17954] [INFO] Listening at: unix:/home/debian/smeet/gunicorn.sock (17954)
[2023-01-16 16:59:41 +0000] [17954] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2023-01-16 16:59:41 +0000] [17957] [INFO] Booting worker with pid: 17957
[2023-01-16 16:59:41 +0000] [17958] [INFO] Booting worker with pid: 17958
[2023-01-16 16:59:41 +0000] [17957] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:59:41 +0000] [17957] [INFO] Worker exiting (pid: 17957)
[2023-01-16 16:59:41 +0000] [17958] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/uvicorn/workers.py", line 66, in init_process
    super(UvicornWorker, self).init_process()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/debian/smeet/smeet-venv/lib/python3.7/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/debian/smeet/app/main.py", line 1, in <module>
    from fastapi import FastAPI
ModuleNotFoundError: No module named 'fastapi'
[2023-01-16 16:59:41 +0000] [17958] [INFO] Worker exiting (pid: 17958)
[2023-01-16 16:59:41 +0000] [17954] [DEBUG] 2 workers
[2023-01-16 16:59:41 +0000] [17954] [WARNING] Worker with pid 17958 was terminated due to signal 15
[2023-01-16 16:59:41 +0000] [17954] [INFO] Shutting down: Master
[2023-01-16 16:59:41 +0000] [17954] [INFO] Reason: Worker failed to boot.
[2023-01-16 17:00:02 +0000] [17989] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fcbfd3272f0>
  on_reload: <function OnReload.on_reload at 0x7fcbfd327400>
  when_ready: <function WhenReady.when_ready at 0x7fcbfd327510>
  pre_fork: <function Prefork.pre_fork at 0x7fcbfd327620>
  post_fork: <function Postfork.post_fork at 0x7fcbfd327730>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fcbfd327840>
  worker_int: <function WorkerInt.worker_int at 0x7fcbfd327950>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fcbfd327a60>
  pre_exec: <function PreExec.pre_exec at 0x7fcbfd327b70>
  pre_request: <function PreRequest.pre_request at 0x7fcbfd327c80>
  post_request: <function PostRequest.post_request at 0x7fcbfd327d08>
  child_exit: <function ChildExit.child_exit at 0x7fcbfd327e18>
  worker_exit: <function WorkerExit.worker_exit at 0x7fcbfd327f28>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fcbfd3430d0>
  on_exit: <function OnExit.on_exit at 0x7fcbfd3431e0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:00:28 +0000] [18027] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fd50bde1510>
  on_reload: <function OnReload.on_reload at 0x7fd50bde1620>
  when_ready: <function WhenReady.when_ready at 0x7fd50bde1730>
  pre_fork: <function Prefork.pre_fork at 0x7fd50bde1840>
  post_fork: <function Postfork.post_fork at 0x7fd50bde1950>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fd50bde1a60>
  worker_int: <function WorkerInt.worker_int at 0x7fd50bde1b70>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fd50bde1c80>
  pre_exec: <function PreExec.pre_exec at 0x7fd50bde1d90>
  pre_request: <function PreRequest.pre_request at 0x7fd50bde1ea0>
  post_request: <function PostRequest.post_request at 0x7fd50bde1f28>
  child_exit: <function ChildExit.child_exit at 0x7fd50bdfc0d0>
  worker_exit: <function WorkerExit.worker_exit at 0x7fd50bdfc1e0>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fd50bdfc2f0>
  on_exit: <function OnExit.on_exit at 0x7fd50bdfc400>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:01:12 +0000] [18089] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f23c03dd6a8>
  on_reload: <function OnReload.on_reload at 0x7f23c03dd7b8>
  when_ready: <function WhenReady.when_ready at 0x7f23c03dd8c8>
  pre_fork: <function Prefork.pre_fork at 0x7f23c03dd9d8>
  post_fork: <function Postfork.post_fork at 0x7f23c03ddae8>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f23c03ddbf8>
  worker_int: <function WorkerInt.worker_int at 0x7f23c03ddd08>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f23c03dde18>
  pre_exec: <function PreExec.pre_exec at 0x7f23c03ddf28>
  pre_request: <function PreRequest.pre_request at 0x7f23c03f80d0>
  post_request: <function PostRequest.post_request at 0x7f23c03f8158>
  child_exit: <function ChildExit.child_exit at 0x7f23c03f8268>
  worker_exit: <function WorkerExit.worker_exit at 0x7f23c03f8378>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f23c03f8488>
  on_exit: <function OnExit.on_exit at 0x7f23c03f8598>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:01:25 +0000] [18103] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f9c3d3ee6a8>
  on_reload: <function OnReload.on_reload at 0x7f9c3d3ee7b8>
  when_ready: <function WhenReady.when_ready at 0x7f9c3d3ee8c8>
  pre_fork: <function Prefork.pre_fork at 0x7f9c3d3ee9d8>
  post_fork: <function Postfork.post_fork at 0x7f9c3d3eeae8>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f9c3d3eebf8>
  worker_int: <function WorkerInt.worker_int at 0x7f9c3d3eed08>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f9c3d3eee18>
  pre_exec: <function PreExec.pre_exec at 0x7f9c3d3eef28>
  pre_request: <function PreRequest.pre_request at 0x7f9c3d40a0d0>
  post_request: <function PostRequest.post_request at 0x7f9c3d40a158>
  child_exit: <function ChildExit.child_exit at 0x7f9c3d40a268>
  worker_exit: <function WorkerExit.worker_exit at 0x7f9c3d40a378>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f9c3d40a488>
  on_exit: <function OnExit.on_exit at 0x7f9c3d40a598>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:01:45 +0000] [18120] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f0ff1bfa6a8>
  on_reload: <function OnReload.on_reload at 0x7f0ff1bfa7b8>
  when_ready: <function WhenReady.when_ready at 0x7f0ff1bfa8c8>
  pre_fork: <function Prefork.pre_fork at 0x7f0ff1bfa9d8>
  post_fork: <function Postfork.post_fork at 0x7f0ff1bfaae8>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f0ff1bfabf8>
  worker_int: <function WorkerInt.worker_int at 0x7f0ff1bfad08>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f0ff1bfae18>
  pre_exec: <function PreExec.pre_exec at 0x7f0ff1bfaf28>
  pre_request: <function PreRequest.pre_request at 0x7f0ff1c160d0>
  post_request: <function PostRequest.post_request at 0x7f0ff1c16158>
  child_exit: <function ChildExit.child_exit at 0x7f0ff1c16268>
  worker_exit: <function WorkerExit.worker_exit at 0x7f0ff1c16378>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f0ff1c16488>
  on_exit: <function OnExit.on_exit at 0x7f0ff1c16598>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:02:54 +0000] [18197] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f5f05959730>
  on_reload: <function OnReload.on_reload at 0x7f5f05959840>
  when_ready: <function WhenReady.when_ready at 0x7f5f05959950>
  pre_fork: <function Prefork.pre_fork at 0x7f5f05959a60>
  post_fork: <function Postfork.post_fork at 0x7f5f05959b70>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f5f05959c80>
  worker_int: <function WorkerInt.worker_int at 0x7f5f05959d90>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f5f05959ea0>
  pre_exec: <function PreExec.pre_exec at 0x7f5f05975048>
  pre_request: <function PreRequest.pre_request at 0x7f5f05975158>
  post_request: <function PostRequest.post_request at 0x7f5f059751e0>
  child_exit: <function ChildExit.child_exit at 0x7f5f059752f0>
  worker_exit: <function WorkerExit.worker_exit at 0x7f5f05975400>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f5f05975510>
  on_exit: <function OnExit.on_exit at 0x7f5f05975620>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:05:13 +0000] [18211] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fa896403730>
  on_reload: <function OnReload.on_reload at 0x7fa896403840>
  when_ready: <function WhenReady.when_ready at 0x7fa896403950>
  pre_fork: <function Prefork.pre_fork at 0x7fa896403a60>
  post_fork: <function Postfork.post_fork at 0x7fa896403b70>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fa896403c80>
  worker_int: <function WorkerInt.worker_int at 0x7fa896403d90>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fa896403ea0>
  pre_exec: <function PreExec.pre_exec at 0x7fa89641f048>
  pre_request: <function PreRequest.pre_request at 0x7fa89641f158>
  post_request: <function PostRequest.post_request at 0x7fa89641f1e0>
  child_exit: <function ChildExit.child_exit at 0x7fa89641f2f0>
  worker_exit: <function WorkerExit.worker_exit at 0x7fa89641f400>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fa89641f510>
  on_exit: <function OnExit.on_exit at 0x7fa89641f620>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:06:09 +0000] [18235] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f581f5118c8>
  on_reload: <function OnReload.on_reload at 0x7f581f5119d8>
  when_ready: <function WhenReady.when_ready at 0x7f581f511ae8>
  pre_fork: <function Prefork.pre_fork at 0x7f581f511bf8>
  post_fork: <function Postfork.post_fork at 0x7f581f511d08>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f581f511e18>
  worker_int: <function WorkerInt.worker_int at 0x7f581f511f28>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f581f52b0d0>
  pre_exec: <function PreExec.pre_exec at 0x7f581f52b1e0>
  pre_request: <function PreRequest.pre_request at 0x7f581f52b2f0>
  post_request: <function PostRequest.post_request at 0x7f581f52b378>
  child_exit: <function ChildExit.child_exit at 0x7f581f52b488>
  worker_exit: <function WorkerExit.worker_exit at 0x7f581f52b598>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f581f52b6a8>
  on_exit: <function OnExit.on_exit at 0x7f581f52b7b8>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:06:32 +0000] [18254] [DEBUG] Current configuration:
  config: gunicorn_conf.py
  wsgi_app: None
  bind: ['51.77.149.238:8000']
  backlog: 2048
  workers: 2
  worker_class: uvicorn.workers.UvicornWorker
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: True
  sendfile: None
  reuse_port: False
  chdir: /home/debian/smeet
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 1000
  group: 1000
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: /home/debian/smeet/access_log
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: /home/debian/smeet/error_log
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: app.main:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f9f56fea8c8>
  on_reload: <function OnReload.on_reload at 0x7f9f56fea9d8>
  when_ready: <function WhenReady.when_ready at 0x7f9f56feaae8>
  pre_fork: <function Prefork.pre_fork at 0x7f9f56feabf8>
  post_fork: <function Postfork.post_fork at 0x7f9f56fead08>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f9f56feae18>
  worker_int: <function WorkerInt.worker_int at 0x7f9f56feaf28>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f9f570040d0>
  pre_exec: <function PreExec.pre_exec at 0x7f9f570041e0>
  pre_request: <function PreRequest.pre_request at 0x7f9f570042f0>
  post_request: <function PostRequest.post_request at 0x7f9f57004378>
  child_exit: <function ChildExit.child_exit at 0x7f9f57004488>
  worker_exit: <function WorkerExit.worker_exit at 0x7f9f57004598>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f9f570046a8>
  on_exit: <function OnExit.on_exit at 0x7f9f570047b8>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2023-01-16 17:06:33 +0000] [18254] [INFO] Starting gunicorn 20.1.0
[2023-01-16 17:06:33 +0000] [18254] [DEBUG] Arbiter booted
[2023-01-16 17:06:33 +0000] [18254] [INFO] Listening at: http://51.77.149.238:8000 (18254)
[2023-01-16 17:06:33 +0000] [18254] [INFO] Using worker: uvicorn.workers.UvicornWorker
[2023-01-16 17:06:33 +0000] [18257] [INFO] Booting worker with pid: 18257
[2023-01-16 17:06:33 +0000] [18257] [INFO] Started server process [18257]
[2023-01-16 17:06:33 +0000] [18257] [INFO] Waiting for application startup.
[2023-01-16 17:06:33 +0000] [18257] [INFO] Application startup complete.
[2023-01-16 17:06:33 +0000] [18258] [INFO] Booting worker with pid: 18258
[2023-01-16 17:06:33 +0000] [18258] [INFO] Started server process [18258]
[2023-01-16 17:06:33 +0000] [18258] [INFO] Waiting for application startup.
[2023-01-16 17:06:33 +0000] [18258] [INFO] Application startup complete.
[2023-01-16 17:06:33 +0000] [18254] [DEBUG] 2 workers
